frontend/src/pages/GuardianProfile.jsx

```
@@ -0,0 +1,409 @@
+      <div className="absolute top-0 left-0 right-0 h-80 bg-gradient-to-r from-blue-500 to-cyan-500 z-0"></div>
+      
+      <div className="max-w-7xl mx-auto px-8 py-8 relative">
+        {/* Edit Mode Toggle Button */}
+        <div className="flex justify-end mb-4 relative z-10">
+          {isEditing ? (
+            <div className="flex gap-2">
+              <button 
+                onClick={() => setIsEditing(false)} 
+                className="bg-slate-200 text-slate-700 px-4 py-2 rounded-lg hover:bg-slate-300 transition-all flex items-center gap-2"
+              >
+                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)